SYCL/ESIMD/vadd_1d.cpp

```diff
@@ -8,7 +8,8 @@
 // REQUIRES: gpu
 // UNSUPPORTED: cuda || hip
 // RUN: %clangxx -fsycl %s -o %t.out
-// RUN: env SYCL_PI_TRACE=-1 %GPU_RUN_PLACEHOLDER %t.out 2>&1 %GPU_CHECK_PLACEHOLDER
+// RUN: env SYCL_PI_TRACE=-1 %GPU_RUN_PLACEHOLDER %t.out 2>&1 %GPU_CHECK_PLACEHOLDER --check-prefix CHECK-W0
+// RUN: env SYCL_PI_TRACE=-1 SYCL_RT_WARNING_LEVEL=1 %GPU_RUN_PLACEHOLDER %t.out 2>&1 %GPU_CHECK_PLACEHOLDER --check-prefix CHECK-W1
```

Comment:
Does it make sense to add a test that would check that stderr and stdout is empty even if there are spills?

Reply:
Surprisingly, I found it difficult to have a test that would reliably give warnings with SYCL_RT_WARNING_LEVEL=1.
After recent updates (I am not sure if that is LLVM/SYCL or GPU RT), majority of tests that printed those warnings stopped doing so.

Regarding 'does it make sense to add a test...' part... 
The current patch checks that SYCL passes the option to GPU RT and that seems sufficient. 
Adding a test that checks stdout/err is empty with -disable-finalizer-msg means to check that GPU RT honors their own flag, which seems a bit excessive to me.
I can though add a test if you think it is needed.

Reply:
mm, maybe you are right.